src/features/ui/MoreNFTOptions/hooks/useMoreOptions.tsx

```diff
@@ -0,0 +1,31 @@
+import { MoreOptions, Option } from '../MoreNFTOptions.types';
+import { OptionType } from '../MoreNFTOptions.constants';
+
+import { OptionLabel } from '../OptionLabel';
+import { IconSend3 } from '@zero-tech/zui/components/Icons';
+
+type UseMoreOptionsReturn = {
+	moreOptions: MoreOptions[];
+};
+
+/**
+ * Returns list of actions, i.e. "transfer", "create token".
+ */
+export const useMoreOptions = (
+	setOption: (option: Option) => void,
+): UseMoreOptionsReturn => {
+	/* More options list - add additional options here */
+	const moreOptions = [
+		{
+			id: OptionType.TRANSFER,
+			label: (
+				<OptionLabel icon={<IconSend3 isFilled />} label="Transfer Ownership" />
+			),
+			onSelect: () => setOption(OptionType.TRANSFER),
+		},
+	];
+
+	return {
+		moreOptions,
+	};
+};
```

Comment:
I'm not sure about using state changes to trigger events - thinking it would be better to define these in the components they're used and linking up proper event handlers.

Reply:
Would `Modal` trigger need updating for this to work correctly?

Reply:
Would still need to set a state object to control which `Modal` is rendered, but I imagine more stuff will be added to More Options which don't necessarily open a modal, so it will be best to define specific event handlers per option rather than setting state for each one.

Reply:
Hey mate, I updated the event handling here. Not sure if its exactly how you expect to see the handling of options - i was confused whether onSelect should be equal to something like `onSelectModal` or remain setting the option here.

Reply:
Happy with whatever works, maybe I'm missing something! Let's get it merged and if it causes issues later we can revisit